docs/cli/ory-proxy.md

```diff
@@ -17,7 +17,7 @@ Run your app and Ory on the same domain using a reverse proxy
 ### Synopsis
 
 This command starts a reverse proxy which must be deployed in front of your application. This proxy works both in development and
-in production, for example when deploying a React, NodeJS, Java, PHP, ... app to a server / the cloud or when developing it
+in production, for example when deploying a React, Node.js, Java, PHP, ... app to a server / the cloud or when developing it
```

Comment:
I'm afraid this will get overwritten when the CLI docs are auto-re-generated again ;( We'd have to adjust it in the CLI code to actually "make it count".